api/rhsm-conduit-api-spec.yaml

```diff
@@ -40,7 +40,7 @@ paths:
         '200':
           description: 'The request for inventory data was successful.'
           content:
-            application/json:
+            application/vnd.api+json:
```

Comment:
Pretty sure this is the case but I just want to verify.  This change will take care of setting the `@Produces` and `@Consumes` annotations on the resource methods?

Reply:
You are correct.